src/pages/login.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+    ChakraProvider,
+    Box,
+    Text,
+    Link,
+    VStack,
+    Code,
+    Grid,
+    Container,
+    Input,
+    InputGroup,
+    InputRightElement,
+    Button,
+   } from '@chakra-ui/react'
+  import React from 'react'
+   
+   export default function Login() {
+    const [show, setShow] = React.useState(false)
+    const handleClick = () => setShow(!show)
+    
+     return (
+       <><Container>
+         <Box borderWidth="1px" borderRadius="1g" p={4}>
+     <Text fontSize="1g" align="center">Log in</Text>
+     <Text fontSize="1g" align="center" pb={4}>Don't have an account? <Link color='teal.500' href='register'>Register</Link></Text>
```

Comment:
Import css styles from external file. Use the chakraUI way of applying styles - sx={stylesObject}